main/SQLHelper.cpp

```diff
@@ -3332,7 +3332,7 @@ void CSQLHelper::ManageExecuteScriptTimeout(int pid, int timeout, bool *stillRun
 }
 #endif
 
-void CSQLHelper::PerformThreadedAction(const _tTaskItem tItem)
+void CSQLHelper::PerformThreadedAction(const _tTaskItem &tItem)
```

Comment:
This is where it crashed last time, please do not undo my changes
Also we do not need to modify the taskItem after the function returns

Reply:
This was automated. I'll look into it.

Reply:
Removed. Easier than figuring out why clang-tidy thinks this should be a reference.